lib/src/rive_core/state_machine_controller.dart

```diff
@@ -429,17 +440,24 @@ class StateMachineController extends RiveAnimationController<CoreContext> {
         }
       }
     }
+    return foundTarget;
   }
 
   void pointerMove(Vec2D position) => _processEvent(
         position,
         hitEvent: ListenerType.move,
       );
 
-  void pointerDown(Vec2D position) => _processEvent(
-        position,
-        hitEvent: ListenerType.down,
-      );
+  void pointerDown(Vec2D position, PointerDownEvent event) {
+    final foundTarget = _processEvent(
+      position,
+      event: event,
+      hitEvent: ListenerType.down,
+    );
+    if (foundTarget) {
+      _recognizer.addPointer(event);
```

Comment:
How does this work, is it sufficient to just call addPointer on the recognizer without registering the recognizer with a GestureDetector?

Reply:
Yes, the recognizer adds itself to the `GestureArenaManager` on `addPointer` (in the parent class `MultiDragGestureRecognizer`)👍